test/index.spec.js

```diff
@@ -0,0 +1,18 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+// import app from '../index.js';
```

Comment:
If this is not in use I believe we should not comment it out but delete it. Please resolve.

Reply:
Un-needed comment has been removed